dictionaries/dictionaries.ts

```diff
@@ -0,0 +1,11 @@
+"server-only";
+import type { Locale } from "./i18n-config";
+
+
+const dictionaries = {
+    en: () => import("./en.json").then((module) => module.default),
+};
+
+export const getDictionary = async (locale: Locale) =>
```

Comment:
typing here is wrong. 

```suggestion
export const getDictionary = async (locale: keyof typeof dictionaries) =>
```